### PR TITLE
Updated Cascadia network

### PR DESCRIPTION
**Pull request by recursivethedev**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -2111,21 +2111,21 @@
...
-    "key": "9000",
+  "6102": {
+    "key": "6102",
     "name": "Cascadia Testnet",
     "shortName": "Cascadia",
-    "chainId": 9000,
+    "chainId": 6102,
     "network": "testnet",
-    "multicall": "0xBd186aca63bD4f5f4eC9E98028461A6A94ac4a12",
+    "multicall": "0x728989819bAD588F193563008E0a03E8cD6a3e4a",
```
Comment (ChaituVR) [inline src/networks.json:2120]: Hi @recursivethedev Can you plz verify and publish code here https://explorer.cascadia.foundation/address/0x728989819bAD588F193563008E0a03E8cD6a3e4a/contracts#address-tabs

File: src/networks.json [hunk 2]
```diff
@@ -2111,21 +2111,21 @@
...
     "rpc": [
-      "https://devnet.cascadia.foundation"
+      "https://testnet.cascadia.foundation"
     ],
     "explorer": {
       "url": "https://explorer.cascadia.foundation"
     },
-    "start": 694000,
-    "logo": "ipfs://QmRgv8ckPdzwHKehJphKDcE3rfuNQaJzjSWq2nuVTTMepZ"
+    "start": 0,
```
Comment (ChaituVR) [inline src/networks.json:2127]:
```suggestion
    "testnet": true,
    "start": 0,
```

Comment (ChaituVR) [inline src/networks.json:2127]:
```suggestion
    "start": 370457,
```

File: src/networks.json (hunk 1 above)
Comment (recursivethedev) [reply src/networks.json:2120]: Hi @ChaituVR , We have verified and publish the contract. Can you please review if everything is good ?

**Comment by ChaituVR** [comment]
@recursivethedev Can you plz look at other comments and also update this branch

File: src/networks.json (hunk 2 above)
Comment (recursivethedev) [reply src/networks.json:2127]: You want us to set the start to 370457 ?

**Comment by recursivethedev** [comment]
I have commented for making sure the changes you asking :)

File: src/networks.json (hunk 2 above)
Comment (ChaituVR) [reply src/networks.json:2166]: Yep

Comment (ChaituVR) [reply src/networks.json:2166]: Yep

Comment (recursivethedev) [reply src/networks.json:2166]: Does it affect something if we don't start at 0 ?

Comment (ChaituVR) [reply src/networks.json]: Yes sir, this way we know when multicall is deployed

Comment (recursivethedev) [reply src/networks.json]: Ok Sir,
I have completed everything and Commited I guess. Right ? 
Anything I need to do more ?

**Comment by ChaituVR** [comment]
Need to update this branch with latest changes :) or allow edits from maintainers
https://user-images.githubusercontent.com/53196251/153501486-176c5603-97dc-4cd3-ae38-e5aa8a37f068.jpeg

**Comment by recursivethedev** [comment]
I clicked Update Branch

it says "At least 1 approving review is required by reviewrs with write access"

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**